linchpin/tests/api/test_init_pass.py

```diff
@@ -76,6 +77,26 @@ def setup_lp_api():
 
 
 @with_setup(setup_lp_api)
+def test_config_extension():
+    test_dict = {
+                'alex': '/path/to/alex',
+                'clint': '/path/to/clint', 
+                'bob': '/path/to/bob'
+            }
+    assert_dict_equal(test_dict, lpa.ctx.cfgs['users'])
+
+@with_setup(setup_lp_api)
+def test_config_override():
```

Comment:
This test doesn't show up in the output you posted. Please repost the data with this test included.